### PR TITLE
Health stat

### PR DESCRIPTION
**Pull request by Spivaka**

### CODE REVIEW

File: Widgets/widget-CPUUsageAvg.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "id": "cpu-usage-avg",
+  "version": -1,
+  "fromVersion": "3.5.0",
+  "name": "CPU usage average",
```
Comment (Bars92) [inline Widgets/widget-CPUUsageAvg.json]: Capitalize

File: Widgets/widget-DiskUsageAvg.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "id": "disk-usage-avg",
+  "version": -1,
+  "fromVersion": "3.5.0",
+  "name": "Disk usage average",
```
Comment (Bars92) [inline Widgets/widget-DiskUsageAvg.json]: Capitalize

File: Widgets/widget-MemoryUsageAvg.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "id": "memory-usage-avg",
+  "version": -1,
+  "fromVersion": "3.5.0",
+  "name": "Memory usage average",
```
Comment (Bars92) [inline Widgets/widget-MemoryUsageAvg.json]: Capitalize

**Comment by meirwah** [comment]
@Spivaka build fails, try to upgrade from master

**Comment by Spivaka** [comment]
tried three times

**Pull request merged**